Snowdog_SolrNavigation/templates/list.phtml

```diff
@@ -86,7 +92,9 @@ $solrDataHelper = $this->helper('Snowdog\SolrNavigation\Helper\Data');
                                     "
                                     type="submit"
                                 >
-                                    <?= __('Add to Cart') ?>
+                                    <span>
+                                        <?= __('Add to Cart') ?>
+                                    </span>
```

Comment:
why did you add span? Can we remove that?

Reply:
b/c of `solrAddToCart`: 

```js
addToCartButton.find('span').text(addToCartButtonTextDefault);
```